Description: Allow building libatomic with dep tracking
Author: Richard Purdie
Origin: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=55930

--- a/src/libatomic/Makefile.am
+++ b/src/libatomic/Makefile.am
@@ -101,7 +101,8 @@
 IFUNC_DEF	= -DIFUNC_ALT=$(PAT_S)
 IFUNC_OPT	= $(word $(PAT_S),$(IFUNC_OPTIONS))
 
-M_DEPS		= -MT $@ -MD -MP -MF $(DEPDIR)/$(@F).Ppo
+@AMDEP_TRUE@M_DEPS		= -MT $@ -MD -MP -MF $(DEPDIR)/$(@F).Ppo
+@AMDEP_FALSE@M_DEPS		= 
 M_SIZE		= -DN=$(PAT_N)
 M_IFUNC		= $(if $(PAT_S),$(IFUNC_DEF) $(IFUNC_OPT))
 M_FILE		= $(PAT_BASE)_n.c
--- a/src/libatomic/Makefile.in
+++ b/src/libatomic/Makefile.in
@@ -331,7 +331,8 @@
 PAT_S = $(word 3,$(PAT_SPLIT))
 IFUNC_DEF = -DIFUNC_ALT=$(PAT_S)
 IFUNC_OPT = $(word $(PAT_S),$(IFUNC_OPTIONS))
-M_DEPS = -MT $@ -MD -MP -MF $(DEPDIR)/$(@F).Ppo
+@AMDEP_TRUE@M_DEPS = -MT $@ -MD -MP -MF $(DEPDIR)/$(@F).Ppo
+@AMDEP_FALSE@M_DEPS = 
 M_SIZE = -DN=$(PAT_N)
 M_IFUNC = $(if $(PAT_S),$(IFUNC_DEF) $(IFUNC_OPT))
 M_FILE = $(PAT_BASE)_n.c
